Lib/turtle.py

```diff
@@ -1937,31 +1937,40 @@ def circle(self, radius, extent = None, steps = None):
         """ Draw a circle with given radius.
 
         Arguments:
-        radius -- a number
-        extent (optional) -- a number
+        radius -- a number (distance)
+        extent (optional) -- a number (angle, in degrees)
         steps (optional) -- an integer
+        
+        With one argument, draw a circle with the given radius. The
```

Comment:
Except that if there is more than one argument, a circle is NOT drawn.  It may be a portion of a circle, or a portion of a polygon.  Try running any proposed documentation past a bright seven-year-old and you'll see some of the rationale for the wording choices I made.  :)

Reply:
Perhaps:
Draw a circle with the given radius when passed the default argument radius.
When additional arguments are given, a circle, arc, polygon, or portion of a polygon may be drawn.